x12-edi/2.5/modules/ROOT/pages/x12-edi-connector-config-topics.adoc

```diff
@@ -44,7 +44,7 @@ NOTE: To use the Functional Acknowledgment schema type (`997` or `999`) it must
 |XML Value |Studio Option
 |formValidation="X12"/"HIPAA_SNIP1"/"HIPAA_SNIP2" |The schema form and validation level can be either X12 or HIPAA. The default is X12.
 |schemas=list of values |List of schema paths the connector can use. These paths can be either a file system or `classpath`.
-|characterEncoding="PLATFORM"/"ASCII"/"ISO8859_1"/"IBM1047"/"UTF8" |The character encoding used for both send and receive messages. The default `PLATFORM` is the Mule platform default encoding
+|characterEncoding="PLATFORM"/"ASCII"/"ISO8859_1"/"IBM1047"/"UTF8" |The character encoding used for both send and receive messages. The default `PLATFORM` is the Mule platform default encoding.
 |stringCharacterSet="BASIC"/"EXTENDED"/"UNRESTRICTED" |Use a substitution character replacement for invalid characters, or reject them as errors if set to `none` or enabled for receive messages in the parser options.  The default is `EXTENDED` for string data.
 |stringSubstitutionChar="?" |Substitution character used to replace invalid characters in string values. The default is `none`.
 |versionIdentifierSuffix="XYZ" |Version identifier code suffix for the group (GS). Setting this value affects both Send and Receive message handling. On the receive side, the GS08 version code must contain X12 schema version (for example, `004010`, `005010`, and so on) followed by this suffix. Not setting this value means the receive-only just verifies that the GS version identifier code starts with the configured schema version. On the Send side, the combination of X12 schema version and this suffix comprises the GS08 value.
```

Comment:
Note that earlier text lowercases send and receive....
In fact, this paragraph mixes the formats.
Standardize.

Reply:
On the receive side, the GS08 version code must contain X12 schema version (for example, `004010`, `005010`, and so on) followed by this suffix.
>
On the receive side, the GS08 version code must contain **the** X12 schema version (for example, `004010`, `005010`, and so on)**,** followed by this suffix.

Reply:
Lines 51-56 hidden